core/base/apiServer.go

```diff
@@ -865,6 +865,11 @@ func handleObjectRequest(orgID string, objectType string, objectID string, write
 //   description: Fetch the objects with expiration time before specified timestamp in RFC3339 format
 //   required: false
 //   type: string
+// - name: deleted
+//   in: query
+//   description: Fetch the object with deleted marked to true
+//   required: false
+//   type: boolean
```

Comment:
This has 3 values, it's not a boolean. if this is a boolean, the API caller will think that no value on the URL will mean false.

Reply:
@dabooz Should I change type to "string"? And in the description, say something like: if specified, value should be "true" or "false". If not specified, result will not be filtered by this field

Reply:
yes, exactly